frontend/src/render.js

```diff
@@ -9,13 +10,17 @@ export function renderValidationErrors(errors) {
                 minDmg: i18n.t('item.' + error.properties[0]),
                 maxDmg: i18n.t('item.' + error.properties[1])
             };
+            count = 2;
         }
         else {
```

Comment:
Wait, why is this function so weird? Why do you need branching on 'mustBeLessThan'? Why do you have `allProps` inside `props`? Isn't it better to put properties inside `props` directly in the same manner as with 'mustBeLessThan'? Then you won't be needing `count` since it would be redundant.